### ISSUE TITLE
Parallele error: object '.doSnowGlobals' not found

### ISSUE THREAD

**Issue by YutingHan317** [privefl/bigsnpr#377]
Dear Florian,

I encountered a problem when I was running LDpred2-grid. The code is as follows:
beta_grid <- snp_ldpred2_grid(corr, df_beta, grid.param, ncores = NCORES)
However, R returned "Error in checkForRemoteErrors(lapply(cl, recvResult)) : 
  19 nodes produced errors; first error: object '.doSnowGlobals' not found"

I've confirmed that the packages like "bigsnpr", "doParallel", "Parallel" were loaded.

How could I solve this? I would appreciate it if you could provide some further advice.

BW,

Yuting

**Comment by privefl** [comment]
Did you close this issue on purpose?
Did you find a solution to your problem? If so, please provide the solution here in case others have the same issue.

**Comment by YutingHan317** [comment]
Thanks for the reminder! I did solve the problem.
Previously, I could correctly run the LDpred-grid on my own PC. However, the above error occurred when I ran the same code on our server.
I first tried to add another package path using ".libPaths()", which did not work.
Then I modified the file "./bashrc" and added a sentence on it, "export R_LIBS='/public/home/hanyuting/Rlib/4.0.3'". Luckily, the problems were solved, though I did not know the reason. 

Kind regards.

Yuting
